h2o-docs/src/product/data-science/gam.rst

```diff
@@ -16,213 +16,136 @@ GAM supports importing and exporting `MOJOs <../save-and-load-model.html#support
-
--  `nfolds <algo-params/nfolds.html>`__: Specify the number of folds for cross-validation. This value defaults to 0 (no cross-validation).
-
--  `seed <algo-params/seed.html>`__: Specify the random number generator (RNG) seed for algorithm components dependent on randomization. The seed is consistent for each H2O instance so that you can create models with the same starting conditions in alternative configurations. This option defaults to -1 (time-based random number).
-
--  `y <algo-params/y.html>`__: (Required) Specify the column to use as the dependent variable.
-
-   -  For a regression model, this column must be numeric (**Real** or **Int**).
-   -  For a classification model, this column must be categorical (**Enum** or **String**). If the family is **Binomial**, the dataset cannot contain more than two levels.
-
--  `x <algo-params/x.html>`__: Specify a vector containing the names or indices of the predictor variables to use when building the model. If ``x`` is missing, then all columns except ``y`` are used.
-
--  `keep_cross_validation_models <algo-params/keep_cross_validation_models.html>`__: Specify whether to keep the cross-validated models. Keeping cross-validation models may consume significantly more memory in the H2O cluster. This option is defaults to true (enabled).
-
--  `keep_cross_validation_predictions <algo-params/keep_cross_validation_predictions.html>`__: Specify whether to keep the cross-validation predictions. This option is defaults to false (not enabled).
+Algorithm-specific parameters
+'''''''''''''''''''''''''''''
 
--  `keep_cross_validation_fold_assignment <algo-params/keep_cross_validation_fold_assignment.html>`__: Enable this option to preserve the cross-validation fold assignment. This option is defaults to false (not enabled).
+-  **gam_columns**: *Required* An array of column names representing the smoothing terms used for prediction. GAM will build a smoother for each specified column. 
 
--  `fold_assignment <algo-params/fold_assignment.html>`__: (Applicable only if a value for **nfolds** is specified and **fold_column** is not specified) Specify the cross-validation fold assignment scheme. The available options are AUTO (which is Random), Random, `Modulo <https://en.wikipedia.org/wiki/Modulo_operation>`__, or Stratified (which will stratify the folds based on the response variable for classification problems). This option defaults to AUTO.
+-  **num_knots**: An array that specifies the number of knots for each predictor specified in ``gam_columns``.
 
--  `fold_column <algo-params/fold_column.html>`__: Specify the column that contains the cross-validation fold index assignment per observation.
+-  **spline_orders**: Order of I-splines/monotone splines used for GAM predictors. For example, ``spline_orders=3`` means a polynomial of order 3 will be used in the splines. If specified, this option must be the same size as ``gam_columns``. Values for ``bs=0`` or ``bs=1`` will be ignored.
```

Comment:
Please change

Order of I-splines/monotone splines

to

Order of I-splines (also known as monotone splines) and NBsplineTypeI M-splines